simcase-game/src/components/Navbar.js

```diff
@@ -4,7 +4,9 @@ import { jsx } from "@emotion/core";
 import { createPortal } from "react-dom";
 import { Dialog } from "@reach/dialog";
 import { navigate } from "@reach/router";
-import ReusableModal from "../views/ReusableModal";
+
+import Modal from "./Modal";
```

Comment:
This is why we need to have a meeting to define all the components in the project. At least have a basic structure from which we will grow. We still have a lot to learn and get used to.